ui/src/scss/_patterns_notifications.scss

```diff
@@ -50,4 +50,9 @@
       margin-left: $spv-outer--small;
     }
   }
+
+  .p-notification + .p-notification,
+  [class^="p-notification--"] + [class^="p-notification--"] {
```

Comment:
Is this something that should go back to Vanilla?

Reply:
I'm not sure tbh. It's not really a bug that's being overwritten, just... the default Vanilla spacing seems a bit too large for MAAS.